openfisca_country_template/parameters/benefits/index.yaml

```diff
@@ -1,3 +1,4 @@
 # This (optional) file defines metadata for the node parameters.benefits in the parameter tree.
 
 description: Social benefits
+documentation: This (optional) file defines metadata for the node parameters.benefits in the parameter tree.
```

Comment:
Same than below, a realistic example would be much better. 
Maybe a generic definition of social benefits? 

Suggestion (extracted from [this Wikipedia page](https://en.wikipedia.org/wiki/Welfare))

> Government support for the citizens and residents of society. They may be provided to people of any income level, as with social security, but usually it is intended to ensure that everyone can meet their basic human needs such as food and shelter.